crates/libcontainer/src/config.rs

```diff
@@ -47,7 +47,11 @@ impl<'a> YoukiConfig {
 
     pub fn load<P: AsRef<Path>>(path: P) -> Result<Self> {
         let path = path.as_ref();
-        let file = fs::File::open(path.join(YOUKI_CONFIG_NAME))?;
+        let config_path = path.join(YOUKI_CONFIG_NAME);
+        if let Err(e) = config_path.try_exists() {
+            return Err(anyhow!("file not exists {}", e));
```

Comment:
Is the only reason to have the extra step to return a better error message? `File::open` will fail when file doesn't exist, so I am not sure the purpose of doing the `try_exist` here.